src/extension/monitor/core/src/test/java/org/geoserver/monitor/ows/MonitorCallbackTest.java

```diff
@@ -211,6 +212,35 @@ public void testWFSGetFeature() throws Exception {
         BBoxAsserts.assertEqualsBbox(expected, data.getBbox(), 0.01);
     }
 
+    @Test
+    @SuppressWarnings("unchecked") // EMF mode without generics
+    public void testWFSGetFeature20() throws Exception {
+        net.opengis.wfs20.GetFeatureType gf = Wfs20Factory.eINSTANCE.createGetFeatureType();
+        org.opengis.filter.Filter f1 = parseFilter("BBOX(the_geom, 40, -90, 45, -60)");
```

Comment:
"org.opengis" does not exist anymore, since GeoTools 30.x/GeoServer 2.24.0.

See the [GeoTools upgrade guide for the 30.x branch](https://docs.geotools.org/stable/userguide/welcome/upgrade.html) and the [GeoServer 2.24.0](https://geoserver.org/announcements/vulnerability/2023/10/15/geoserver-2-24-0-released.html) release announcement.

Reply:
Updated to `org.geotools.api`